### PR TITLE
Added DoD specific docs for Iron Bank Repo1

### PR DESCRIPTION
**Pull request by jordanh**
Going with "some is better than perfect" on these edits. Open to comments/edits (edits preferred as addn'l commits!)

In the future, we'll have a federal offering manager who'll produce better version of these docs. The goal with these edits to have _something_ that says, "hey you want to use this, contact us here" if they pull this down from Iron Bank.

Also removed old .md files
Minor tweaks to root README.MD and LICENSE

### CODE REVIEW

**Comment by jordanh** [comment]
@adaniels-parabol after you've had a chance to look at this, please pass it to @mattkrick for Maintainer review

File: docs/alternative-licenses/us-department-of-defense/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+- Fill out the form with values from your `.env`. Set the host to `postgres`
+
+## Have feedback, ideas or feature requests?
+
+Please review our [Discussions](https://github.com/ParabolInc/parabol/discussions) to see if there's already a similar suggestion, and if not please feel free to [start a new one](https://github.com/ParabolInc/parabol/discussions/new).
+
+## Helpful VSCode extensions
+
+We use [GraphQL](https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql) for IntelliSense and syntax highlighting.
```
[the remark this answers is not in the document]
Comment (adaniels-parabol) [reply docs/alternative-licenses/us-department-of-defense/README.md:101]: Eh, I don't want to hold this up, possible discuss in future a revision.

File: docs/alternative-licenses/us-department-of-defense/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# Parabol Hardened Container for Platform One Iron Bank
```
[the remark this answers is not in the document]
Comment (adaniels-parabol) [reply docs/alternative-licenses/us-department-of-defense/README.md:1]: Eh, I don't want to hold this up, possible discuss in future a revision.

**Review by adaniels-parabol — APPROVED** [review]

**Comment by adaniels-parabol** [comment]
added @mattkrick for review

**Comment by mattkrick** [comment]
ah shucks, just updated the readme yesterday 😆 
the DOD-specific readme is based off the old version, but i don't that matters much

**Pull request merged**